src/OpenLoco/src/Windows/ScenarioOptions.cpp

```diff
@@ -469,7 +462,8 @@ namespace OpenLoco::Ui::Windows::ScenarioOptions
                 widgets[widx::time_limit_value].type = WidgetType::textbox;
                 widgets[widx::time_limit_value_down].type = WidgetType::button;
                 widgets[widx::time_limit_value_up].type = WidgetType::button;
-                _commonFormatArgs[3] = Scenario::getObjective().timeLimitYears;
+                args.skip(2);
```

Comment:
I don't think this is correct. It needs to be relative to the above pushes that varied in size from 2 and 1 slots. This was to go in the 3 slot regardless. I think just add a skip 1 on L436 .

Reply:
You're right this skip isn't enough in the case of `ObjectiveType::performanceIndex`. Thinking about this, though, I think it's more complicated. `_commonFormatArgs` was defined as an uint16_t array in this file, while FormatArguments uses std::bytes internally. So position 3 here will be equivalent to position 6 in the new situation.

So I'm inclined to leave this skip(2) here, while adding another skip(2) on line 436. Thoughts?

Reply:
Yes you are quite right should be two skip(2)'s I thought skip was in words as well.